src/server/routes.go

```diff
@@ -93,10 +94,24 @@ func (s *Server) handleManifest(c *router.Context) {
 				"sizes": "64x64",
 				"type":  "image/png",
 			},
+			{
+				"src":   s.BasePath + "/static/graphicarts/favicon-144.png",
```

Comment:
is png a must? wouldn't svg be sufficient? why 144 (out of curiosity)?

Reply:
That's what my browser (Vivaldi or Chromium-based browsers) requires for an installable PWA. You can turn on the developer console and switch to tab "Application" and see how it goes.

![image](https://github.com/nkanaev/yarr/assets/27758849/41765c2a-8713-42b6-b152-ecc35e58b3c2)